orc8r/cloud/go/services/orchestrator/servicers/builder_servicer_test.go

```diff
@@ -32,6 +33,93 @@ import (
 	mconfig_protos "magma/orc8r/lib/go/protos/mconfig"
 )
 
+func testUtilForTestSharedConfig(t *testing.T, orc8rVersion string) {
+	expectedDefaultJitteredSyncIntervalGW1 := uint32(71)
+	nw := configurator.Network{ID: "n1", Configs: map[string]interface{}{
+		orc8r.NetworkSentryConfig: &models.NetworkSentryConfig{
+			SampleRate:   swag.Float32(0.75),
+			UploadMmeLog: true,
+			URLPython:    "https://www.example.com/v1/api",
+			URLNative:    "https://www.example.com/v1/api",
+		},
+	}}
+	gw := configurator.NetworkEntity{
+		Type: orc8r.MagmadGatewayType,
+		Key:  "gw1",
+		Config: &models.MagmadGatewayConfigs{
+			AutoupgradeEnabled:      swag.Bool(true),
+			AutoupgradePollInterval: 300,
+			CheckinInterval:         60,
+			CheckinTimeout:          10,
+			DynamicServices:         []string{},
+			FeatureFlags:            map[string]bool{},
+		},
+	}
+	graph := configurator.EntityGraph{
+		Entities: []configurator.NetworkEntity{gw},
+	}
+
+	expected := map[string]proto.Message{
```

Comment:
Since we only care about testing the Orc8rVersion in this test, we can grab just that specific mconfig and compare it, rather than specifying and comparing the entire map